### DISCUSSION TITLE
Wolfberry

### DISCUSSION THREAD

**Discussion by john30** [john30/ebusd#953] [Announcements]
This is the 14th release of ebusd 3 and the 23rd (="W", therefor [Wolfberry](https://en.wikipedia.org/wiki/Wolfberry)) since ebusd was completely reworked.

The major new features are value lists for MQTT integration, update check for v5 devices, workarounds for libssl plus a bunch of bug fixes.

See [ChangeLog](https://github.com/john30/ebusd/blob/master/ChangeLog.md) and [Wiki](https://github.com/john30/ebusd/wiki) for details.

Debian 12 (Bookworm), 11 (Bullseye), and 10 (Buster) install packages for amd64, 386, armv7 (Raspberry Pi 2/3/4), and arm64 (Raspberry Pi 3/4 64 bit) are attached and will also be made available soon in the [Debian repository as described here](https://github.com/john30/ebusd-debian/blob/master/README.md).

For convenience, an armv6l (Raspberry Pi 1 and Zero/Zero W) Buster image is attached here as well (not part of the Debian repository though).

Variants of each binary with MQTT support have an additional "mqtt1" (for libmosquitto1) suffix in the name, whereas those without don't.

For Arch Linux, simply run `makepkg` in the `contrib/archlinux/ebusd/` directory.

For Alpine Linux, run `apk --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community add ebusd`.

The latest release is also available as [multi-architecture Docker image on the hub](https://hub.docker.com/r/john30/ebusd) with tag ebusd:latest.

<hr /><em>This discussion was created from the release <a href='https://github.com/john30/ebusd/releases/tag/23.2'>Wolfberry</a>.</em>

**Comment by xpavli44** [comment]
https://github.com/john30/ebusd/issues/958
